lbry/lbry/conf.py

```diff
@@ -170,6 +170,11 @@ def deserialize(self, value):
                 'amount': float(value['amount']),
             }
         if isinstance(value, str):
+            if not value:
+                return None
```

Comment:
Can you combine this into the initial `if value is None:` and just make that one `if value:`.

Reply:
Yes. That works and is a lot cleaner